lib/ReactViews/Preview/Description.jsx

```diff
@@ -180,23 +183,31 @@ const Description = observer(
               </Choose>
             </If>
 
-            <If condition={catalogItem.metadataUrl}>
-              <h4 className={Styles.h4}>{t("description.metadataUrl")}</h4>
-              <p>
-                <a
-                  href={catalogItem.metadataUrl}
-                  target="_blank"
-                  rel="noopener noreferrer"
-                  className={Styles.link}
-                  css={`
-                    color: ${p => p.theme.colorPrimary};
-                  `}
-                >
-                  {catalogItem.metadataUrl}
-                </a>
-              </p>
+            <If
+              condition={
+                catalogItem.metadataUrls && catalogItem.metadataUrls.length > 0
+              }
+            >
+              <h4 className={Styles.h4}>{t("description.metadataUrls")}</h4>
+              <For each="metadataUrl" index="i" of={catalogItem.metadataUrls}>
+                <Box paddedVertically>
```

Comment:
There is a console warning for not passing a key for the list child, add a `key={metadataUrl.url}` for the `Box` here?

Reply:
Done!